scripts/create_changeset.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+VALUES_FILE_PATH="./stable/ksoc-plugins/values.yaml"
+CHART_FILE_PATH="./stable/ksoc-plugins/Chart.yaml"
+
+HELM_CHART_VERSION=$(yq e ".version" "$CHART_FILE_PATH")
+HELM_CHART_REPO="ksoc-labs/ksoc-plugins"
+
+RELEASE_NOTES_URL="https://github.com/ksoclabs/ksoc-plugins-helm-chart"
+
+AWS_MARKETPLACE_PRODUCT_ID="prod-m7tlrvfq6yjzu"
+
+
+CONTAINER_IMAGES="709825985650.dkr.ecr.us-east-1.amazonaws.com/ksoc-labs/falco-no-driver:0.36.2
+709825985650.dkr.ecr.us-east-1.amazonaws.com/ksoc-labs/falcoctl:0.6.2\n"
+
+PRODUCT_TITLE=$(aws marketplace-catalog describe-entity --entity-id "${AWS_MARKETPLACE_PRODUCT_ID}" --catalog AWSMarketplace --region us-east-1 | jq -r '.Details | fromjson | .Description.ProductTitle')
+
+images=""
+detected_images=$(yq e 'keys | .[]' "$VALUES_FILE_PATH")
+
+for image in $detected_images; do
+    repo=$(yq e ".${image}.image.repository" "$VALUES_FILE_PATH")
+    tag=$(yq e ".${image}.image.tag" "$VALUES_FILE_PATH")
+
+    # Check if both repository and tag are not null or empty
+    if [ -n "$repo" ] && [ "$repo" != "null" ] && [ -n "$tag" ] && [ "$tag" != "null" ]; then
+        images+="${repo}:${tag}\n"
+    fi
+done
+
+# Appendthe images found in values.yaml with our Falco images
```

Comment:
Nit: `Append the`

Reply:
Actually, do we have to mention Falco here?

Reply:
We do. 😞 All images have to be hosted within AWS, so I manually copied them over for now. When Falco gets replaced, we can remove it.